orderbook/src/database/orders.rs

```diff
@@ -325,6 +360,8 @@ impl OrdersQueryRow {
                     .map_err(|_| anyhow!("signature has wrong length"))?,
             ),
             signing_scheme: self.signing_scheme.into(),
+            sell_token_balance: self.balance_from.into(),
+            buy_token_balance: self.balance_to.into(),
```

Comment:
Related to :point_up:: it feels weird that this is the only field that doesn't have consistent naming between SC order properties, `model::Order` fields, `database::Order` fields and database column names. I just feel like it may cause confusion in the future.